### PR TITLE
feat(embodied): add opensora world simulator

### PR DESCRIPTION
**Pull request by jzndd**
RL training with Opensora world simulator 

### Description
This PR integrates the Opensora World Model into the RLinf framework, providing a working end-to-end instantiation that can already run successfully.

### Motivation and Context
Opensora serves as the first fully functional example, validating the pipeline and allowing subsequent models to be dropped into the same interface.

### How has this been tested?
+ Successfully executed Opensora-based world model inside RLinf environment.
+ End-to-end rollout tested with real chunking, inference, and reward extraction.
+ Note: This is a new feature and CI coverage has not been added yet.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation update (Document-only update)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: rlinf/envs/world_model/world_model_opensora_env.py [hunk 1]
```diff
@@ -0,0 +1,890 @@
...
+        episode_info = {}
+        self.returns += step_reward
+        # Update success_once based on terminations
+        if isinstance(terminations, torch.Tensor):
+            self.success_once = self.success_once | terminations
+        else:
+            terminations_tensor = torch.tensor(
+                terminations, device=self.device, dtype=torch.bool
+            )
+            self.success_once = self.success_once | terminations_tensor
```
[the remark this answers is not in the document]
Comment (jzndd) [reply rlinf/envs/world_model/world_model_opensora_env.py:226]: Not Necessary

**Comment by QuanluZhang** [comment]
@guozhen1997 @andylin-hao  as primary reviewer

**Comment by andylin-hao** [comment]
Hi @jzndd, I've improved the install script and docs, and added docker and CI in https://github.com/jzndd/RLinf/pull/1. Could you check and merge it?
